Assets/PlayerMovement.cs

```diff
@@ -34,6 +36,12 @@ void Start()
         canvasPosition = GameObject.Find("Canvas").transform.position;
         col = GetComponent<Collider2D>();
         UpdateSprite();
+
+        backButton = GameObject.Find("MenuButton");
+        backButton.transform.SetPositionAndRotation(new Vector3((Screen.width - (Screen.width / 6)), (Screen.height - (Screen.height / 10)), 0), Quaternion.identity);
```

Comment:
Kinda weird to have the back button and score positioned in the PlayerMovement code, maybe move this to the LevelController?

Reply:
Moved them both to LevelController. Tried to move Score to the Score.cs but the transform applies on the lose screen too, oh well.